src/OpenTelemetry.Exporter.Console/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Released 2021-Nov-29
   the `ConsoleMetricExporter` to export either manually or periodically.
   ([#2648](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2648))
 
+* ActivityExporter exports Activity Links.
```

Comment:
nit: `ConsoleActivityExporter`

Reply:
```suggestion
* Added `Activity.Links` support to `ConsoleActivityExporter`.
```

Reply:
This is supposed to be under "Unreleased" section?

Reply:
fixed.

Reply:
Reopen.